src/main/java/Assignment8.java

```diff
@@ -0,0 +1,11 @@
+//Printing middle number
+public class Assignment8 {
+    public static void main(String[] args){
```

Comment:
Instead of writing this code in void main, lets define a function for this. and lets create test in src/test/* package which tests your function